notebooks/xvcfmerge-array-input.py

```diff
@@ -81,12 +85,22 @@ def update_row(table: str, row_name: str, updates: dict):
                                                                       "drs://dg.4503/aba6b011-2ab4-4739-beb4-c1eeaee60c74"],
                                                               output=f"{bucket}/merged/drs_combined_b.vcf.gz"))
 
+with callysto.Cell("markdown"):
+    """
+    ## Option B: Prepare the merge workflow input data table for VCFs already in bucket
+    This workflow preparation uses VCFs that are present in your workspace bucket. Results will be placed in your workspace bucket.
+
+    Make sure that they follow the following format:
+    `gs://[your-bucket's-name]/vcfsa/chr1.vcf.gz`
+    `gs://[your-bucket's-name]/vcfsa/chr2.vcf.gz`
+    `..`
+    `gs://[your-bucket's-name]/vcfsb/chr1.vcf.gz`
+    `gs://[your-bucket's-name]/vcfsb/chr2.vcf.gz`
```

Comment:
There are no requirements on the format of the Google URLs.

For instance, this is also valid:
```
`gs://[your-bucket's-name]/argle/chr1.vcf.gz`
`gs://[your-bucket's-name]/names/or/something/frank.vcf.gz`
`gs://[your-bucket's-name]/bob.vcf.gz`
```

Whatever the bucket keys used to reference the VCFs, the input table must be prepared such that each row involves a single Chromosome.

Reply:
Right, but doesn't the reference table _as written_ require the vcfsa vcfsb format?

Reply:
There is no such requirement, it's just an example that I hope users can adapt to their needs.